tests/test_snopt_bugfix.py

```diff
@@ -21,35 +21,6 @@ def objfunc(xdict):
     funcs = {}
 
     funcs["obj"] = (x - 3.0) ** 2 + x * y + (y + 4.0) ** 2 - 3.0
-    conval = -x + y
-    funcs["con"] = conval
-
-    fail = False
-    return funcs, fail
-
-
-def objfunc_no_con(xdict):
```

Comment:
Nice catch, I assume the new check had these tests failing right away, right?

Reply:
Yes you're right